soccer/planning/TrapezoidalPath.cpp

```diff
@@ -63,8 +66,55 @@ bool TrapezoidalPath::hit(const Geometry2d::ShapeSet& obstacles,
 
 std::unique_ptr<Path> TrapezoidalPath::subPath(RJ::Seconds startTime,
                                                RJ::Seconds endTime) const {
-    debugThrow("This function is not implemented");
-    return nullptr;
+    // Check for valid arguments
```

Comment:
I'm 60% sure this was implemented but never used, this should be removed from the PR assuming nothing crashes

Reply:
TrapezoidalPath as a whole or only subPath?

Reply:
Only the changes. This is on the capture branch

Reply:
Edit, this is actually used.